### PR TITLE
Rename dms* -> catalyst*

### PR DESCRIPTION
**Pull request by thomshutt**
See this thread for context on the rename: https://discord.com/channels/423160867534929930/991388659699949669/1004599546933948486

### CODE REVIEW

**Comment by hjpotter92** [comment]
There might be some similar changes needed in gh-workflow?

**Comment by thomshutt** [comment]
> There might be some similar changes needed in gh-workflow?

Good call, for some reason my recursive `grep` didn't go into the `.github`

File: .github/workflows/test.yaml
Comment (hjpotter92) [inline .github/workflows/test.yaml:51]: this would apply only when the project gets renamed :|

try setting a `CODEOWNERS` file and see if that grants you access to the project :smile:

**Review by hjpotter92 — APPROVED** [review]
lgtm

File: .github/workflows/test.yaml
Comment (thomshutt) [reply .github/workflows/test.yaml:51]: Yeah, I was going to wait for the rename before merging

**Review by victorges — APPROVED** [review]

**Pull request merged**
